src/plugins/intel_gpu/src/graph/graph_optimizer/compile_graph.cpp

```diff
@@ -20,9 +20,8 @@ using namespace cldnn;
 
 void compile_graph::run(program& p) {
     OV_ITT_SCOPED_TASK(itt::domains::CLDNN, "CLDNN::pass::CompileGraph");
-    size_t order_idx = 0;
     for (auto& node : p.get_processing_order()) {
-        node->set_unique_id(std::to_string(order_idx++));
+        node->set_unique_id();
```

Comment:
[random spot] Could you add a test for such case? I think it can be based on that one: https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/gpu/behavior/cache.cpp#L26

Reply:
Okay, I'll add.

Reply:
Added